src/com/lftechnology/java/training/alina/jdbc/dbutils/DbFacade.java

```diff
@@ -0,0 +1,117 @@
+package com.lftechnology.java.training.alina.jdbc.dbutils;
+
+import java.sql.Date;
+import java.sql.DriverManager;
+import java.sql.ParameterMetaData;
+import java.sql.SQLException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+
+import com.lftechnology.java.training.alina.jdbc.domain.Database;
+
+/**
+ * DbFacade consists of database connection related functionalities
+ * 
+ * @author Alina Shakya <alinashakya@lftechnology.com>
+ */
+public class DbFacade {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)